drivers/rptun/rptun.c

```diff
@@ -953,7 +953,7 @@ int rptun_initialize(FAR struct rptun_dev_s *dev)
       goto err_driver;
     }
 
-  snprintf(arg1, 16, "0x%" PRIxPTR, (uintptr_t)priv);
+  snprintf(arg1, 19, "0x%" PRIxPTR, (uintptr_t)priv);
```

Comment:
```suggestion
  snprintf(arg1, sizeof(arg1), "0x%" PRIxPTR, (uintptr_t)priv);
```
and same for `name` above

Reply:
Right, will re-post